gquant/dataframe_flow/_node_flow.py

```diff
@@ -4,14 +4,17 @@
 import dask
 import cudf
 import dask_cudf
+import copy
 
 from .taskSpecSchema import TaskSpecSchema
 from .portsSpecSchema import PortsSpecSchema
 
-OUTPUT_ID = 'f291b900-bd19-11e9-aca3-a81e84f29b0f_uni_output'
+# OUTPUT_ID = 'f291b900-bd19-11e9-aca3-a81e84f29b0f_uni_output'
+OUTPUT_ID = 'collector_id_fd9567b6'
+OUTPUT_TYPE = 'Output Collector'
```

Comment:
Best practice is to avoid having a space in a string identifier. Could you change it to `Output_Collector` using an underscore?

I found this identifier in typescript code, taskgraphs and some of the notebooks.

Reply:
https://github.com/yidong72/gQuant/blob/757d027d0a87cacb46b51a298716fd27ee0c77d8/docker/build.sh#L28

Typo. Please fix to:
```
        OS_STR="ubuntu20.04"
```

https://github.com/yidong72/gQuant/blob/757d027d0a87cacb46b51a298716fd27ee0c77d8/docker/build.sh#L53-L54
Typo. Please fix echo to:
```
        echo "CUDA 11.0 is selected"
```

https://github.com/yidong72/gQuant/blob/757d027d0a87cacb46b51a298716fd27ee0c77d8/docker/build.sh#L90-L92
These three lines should come after the copying of gQuant to local directory, because the `grep version gQuant/setup.py` looks inside that gQuant subdirectory. Change to:
```
mkdir -p gQuant
cp -r ../gquant ./gQuant
cp -r ../task_example ./gQuant
cp -r ../modules ./gQuant
cp -r ../taskgraphs ./gQuant
cp ../setup.cfg ./gQuant
cp ../setup.py ./gQuant
cp ../LICENSE ./gQuant
cp ../download_data.sh ./gQuant
cp ../gquantrc ./gQuant
rsync -av --progress ../notebooks ./gQuant --exclude data --exclude .cache --exclude many-small --exclude storage --exclude dask-worker-space --exclude __pycache__
rsync -av --progress ../gquantlab ./gQuant --exclude node_modules 

gquant_ver=$(grep version gQuant/setup.py | sed "s/^.*version='\([^;]*\)'.*/\1/")
CONTAINER="nvidia/cuda:${CUDA_STR}-runtime-${OS_STR}"
D_CONT=${D_CONT:="gquant/gquant:${gquant_ver}-${CUDA_STR}_${OS_STR}_${RAPIDS_VERSION}_${MODE_STR}"}

```

Reply:
fixed